runtime/v2/runc/container.go

```diff
@@ -506,3 +477,25 @@ func (c *Container) HasPid(pid int) bool {
 	}
 	return false
 }
+
+func loadProcessCgroup(ctx context.Context, pid int) (cg interface{}, err error) {
+	if cgroups.Mode() == cgroups.Unified {
+		g, err := cgroupsv2.PidGroupPath(pid)
+		if err != nil {
+			logrus.WithError(err).WithError(err).Errorf("loading cgroup2 for %d", pid)
```

Comment:
Accidentally `WithError` twice here; also line 490 and 496.

Reply:
ah good catch. must be made when I change `log` to `logrus`. will fix

Reply:
fixed